packages/moda-sdk/src/core/index.ts

```diff
@@ -1 +1,2 @@
 export const core = 'core';
+export * from '../profile';
```

Comment:
We don't want to export `profile` from `core`. Core is a separate "module" / export path.